html/controls/lt/lt-sheet.js

```diff
@@ -246,27 +246,34 @@ function prepareFieldingEditor(teamId) {
 	}		
 	
 	function processSkater(k, v) {
-		var match = (k || "").match(/Skater\(([^\)]+)\)\.Number/);
+		var match = (k || "").match(/Skater\(([^\)]+)\)\.([^\.]+)/);
```

Comment:
Don't forget you can use the new k.Skater etc.

Reply:
I left that for when the file is rewritten to follow your approach.